samples/log_callback/host/host.c

```diff
@@ -65,10 +65,16 @@ void customized_log(
         message);
 }
 
+void host_transfer_logs_to_file(const char* modified_log, size_t size)
+{
+    FILE* enc_logfile = fopen("./oe_enclave_out.txt", "w");
+    fprintf(enc_logfile, "%.*s", (int)size, modified_log);
```

Comment:
fclose after this.

Reply:
Added fclose for enc_logfile as part of label exit: in main().